source/viz.md

```diff
@@ -1047,30 +1116,28 @@ glue('islands_bar_sorted', islands_bar_sorted, display=True)
 ```
 
 :::{glue:figure} islands_bar_sorted
-:figwidth: 700px 
+:figwidth: 700px
 :name: islands_bar_sorted
 
 Bar plot of size for Earth's largest 12 landmasses.
 :::
 
 
-
-
-The plot in {numref}`islands_bar_sorted` is definitely clearer now, 
-and allows us to answer our question 
-("are the top 7 largest landmasses continents?") in the affirmative. 
-But the question could be made clearer from the plot 
+The plot in {numref}`islands_bar_sorted` is definitely clearer now,
+and allows us to answer our question
+("Which are the top 7 largest landmasses continents?") in the affirmative.
+But the question could be made clearer from the plot
 by organizing the bars not by alphabetical order
-but by size, and to color them based on whether they are a continent. 
-The data for this is stored in the `landmass_type` column. 
-To use this to color the bars, 
+but by size, and to color them based on whether they are a continent.
+The data for this is stored in the `landmass_type` column.
+To use this to color the bars,
 we use the `color` argument to color the bars according to the `landmass_type`
 
-To organize the landmasses by their `size` variable, 
+To organize the landmasses by their `size` variable,
 we will use the `altair` `sort` function
 in encoding for `y` axis to organize the landmasses by their `size` variable, which is encoded on the x-axis.
 To sort the landmasses by their size(denoted on `x` axis), we use `sort='x'`. This plots the values on `y` axis
-in the ascending order of `x` axis values. 
+in the ascending order of `x` axis values.
 
 We do this here so that the largest bar will be closest to the axis line,
 which is more visually appealing.
```

Comment:
Note box below here is not necessary. Just put it as part of the text that you can do descending by `-x`